misc/dist-test-cases

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "EXTRA_DIST = misc/units \\" > "$1"
+find Tmain Units -type f | sed -e 's/$/\\/' >> "$1"
+echo '$(NULL)' >> "$1"
```

Comment:
This is about autogen.sh. So I will use git list-files for listing files under Units and Tmain.